### DISCUSSION TITLE
How to effectively process multiple .pdb files?

### DISCUSSION THREAD

**Discussion by zyang20** [Electrostatics/pdb2pqr#226] [Q&A]
Hi all, does anyone know how to do a batch processing of multiple .pdb files? the web server only allows one at a time.
Thanks

**Comment by sobolevnrm** [comment]
I would recommend downloading the PB2PQR binary with `pip install pdb2pqr` and using that.

**Comment by zyang20** [comment]
I did that in Jupyer notebook. But I couldn't figure out how to launch the software. There are no clear instructions. Anyone could give me instructions step by step? Thanks a lot.

**Comment by sobolevnrm** [comment]
Pip installs the command-line program `pdb2pqr30` that you can use to run PDB2PQR--that is the preferred mode of execution (not via a notebook).  However, pip also installs the `pdb2pqr` module from which you could import [`main_driver`](https://github.com/Electrostatics/pdb2pqr/blob/810e168740917c6a01e392444460c0c38fcce716/pdb2pqr/main.py#L730) and manually pass the desired argument namespace returned by [`build_parser`](https://github.com/Electrostatics/pdb2pqr/blob/810e168740917c6a01e392444460c0c38fcce716/pdb2pqr/main.py#L38).  

I'm going to convert this issue to a discussion since others may have the same question.

**Reply by zyang20** [reply]
Thanks sobolevnrm for the reply. Actually I  did 'pip install pdb2pqr' using the terminal launched from Annaconda (this was Windows 10). I will give it a try in Mac or Linux terminal. Thanks
